pylintrc

```diff
@@ -3,7 +3,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
-extension-pkg-whitelist=
+extension-pkg-whitelist=rucio.common.hash_algorithms
```

Comment:
Which extensions are loaded here? I don't see them. :sweat_smile:

Reply:
From what I could find on the Web this is the only way to convince pylint not to complain about symbols from dynamically loaded modules. Not that it works as things stand, CI runs clearly show pylint keeps complaining...